conda_smithy/configure_feedstock.py

```diff
@@ -340,9 +340,6 @@ def _collapse_subpackage_variants(list_of_metas):
     _trim_unused_zip_keys(used_key_values)
     _trim_unused_pin_run_as_build(used_key_values)
 
-    print("top_level_loop_vars", top_level_loop_vars)
```

Comment:
I've found this logging quite useful when running smithy locally

Reply:
They should be optional and enabled using a flag. I've restored them for now